warehouse/query-core/src/main/java/datawave/query/util/MetadataHelperCacheManagementListener.java

```diff
@@ -24,16 +24,16 @@
     private static final Logger log = Logger.getLogger(MetadataHelperCacheManagementListener.class);
     
     private String zookeepers;
-    private MetadataHelper metadataHelper;
+    private TypeMetadataHelper typeMetadataHelper;
```

Comment:
Pardon my ignorance here, but it appears to me that MetadataHelper and TypeMetadataHelper are entirely different classes. Why this change (and the same change in the MetadatHelperUpdateHdfsListener class)?

Reply:
What goes into hdfs comes from the TypeMetadatahelper. The MetadataHelper has a TypeMetadataHelper, but this change it to make the connection more direct. This way, the code that uses the TypeMetadataHelper will not be encumbered by the other baggage/dependencies that come with the MetadataHelper instance tree.
The metadataTableName is no longer stateful in the TypeMetadataHelper. One instance of the TypeMetadataHelper may be used to update the hdfs cache of all of the metadata tables.
The metadataTableName remains stateful in the MetadataHelper, because the MetadataHelper is created and initialized for a specific metadata table, and that should not change.